atoMEC/writeoutput.py

```diff
@@ -510,16 +525,16 @@ def density_to_csv(rgrid, density, filename):
     """
     if config.spindims == 2:
         headstr = (
-            "r"
-            + 7 * " "
-            + "n^up_b"
+            "r (a_0)"
             + 4 * " "
-            + "n^up_ub"
+            + "n^u (orbs)"
             + 3 * " "
-            + "n^dw_b"
-            + 4 * " "
-            + "n^dw_ub"
+            + "n^u (ideal)"
+            + 2 * " "
+            + "n^d (orbs)"
             + 3 * " "
+            + "n^d (ideal)"
+            + 1 * " "
         )
```

Comment:
I do not understand your intention here. Why is it necessary to assemble the string from small parts? And why so complicated?
Suggestion, something like
```python
headstr = r" ".join(["r (a_0)", "n^u (orbs)", "n^u (ideal)", "n^d (orbs)", "n^d (ideal)"])
```

Reply:
Haha, well I wanted the headstrings to line up with the number underneath (see e.g. contrasting images below) and didn't immediately know / was too lazy to investigate how to do this the 'proper' way (I guess with string formatting).

I'll look into it unless you know the solution easily? Alternatively, we could just switch to comma separated values (and likewise in the headstring) in which case the lack of alignment wouldn't bother me so much.

![output_2](https://user-images.githubusercontent.com/18050753/150380416-235b0165-fd53-494d-9ff4-8b13a46345ee.png)
![output_1](https://user-images.githubusercontent.com/18050753/150380420-360d124c-4b77-4d12-8a02-84ed8e157026.png)

Reply:
Ok look at new commit 23c5826 and see what you think

Reply:
See my comment here: https://github.com/atomec-project/atoMEC/pull/123/commits/23c58d6b324599e32f1f3af2b2fcd422c29d50f4#r789684125

Reply:
> Please be aware, that some of my suggestions introduced pep8 style issues, see the failed pipeline checks!

Ah yes all sorted now!